Tasks/AzureKeyVaultV1/task-runner.ts

```diff
@@ -0,0 +1,41 @@
+import tl = require("azure-pipelines-task-lib/task");
+import path = require("path");
+
+import keyVaultTaskParameters = require("./models/KeyVaultTaskParameters");
+import keyVault = require("./operations/KeyVault");
+
+export class TaskRunner {
+    public static async runInContext(isPreJobContext: boolean): Promise<void> {
+        const runAsPreJob: boolean = tl.getBoolInput('RunAsPreJob', true);
+        if (runAsPreJob === isPreJobContext) {
+            this.run();
+        } else {
+            tl.setResult(tl.TaskResult.Skipped, "Skipped according to the task's configuration.");
```

Comment:
Nit: include the runAsPreJob value here: `"Skipped according to the task's configuration. RunAsPreJob: " + runAsPreJob`

Reply:
Thank you for the review! I have added this information.